commons/src/main/java/io/aiven/kafka/connect/common/source/input/ParquetTransformer.java

```diff
@@ -43,50 +40,48 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ParquetTransformer extends Transformer<GenericRecord> {
-
+/**
+ * A transformer to read parquet files. The input file is read one Avro record at a time. This implementation creates a
+ * temporary file to read the data from, and removes it at the end of processing.
+ */
+public class ParquetTransformer extends Transformer {
+    /** The AvroData to read with */
     private final AvroData avroData;
-
+    /** The logger for this transform */
     private static final Logger LOGGER = LoggerFactory.getLogger(ParquetTransformer.class);
 
+    /**
+     * Constructs a ParquetTransformer that reads with the AvroData.
+     *
+     * @param avroData
+     *            The AvroData to read with.
+     */
     ParquetTransformer(final AvroData avroData) {
         super();
         this.avroData = avroData;
     }
 
     @Override
-    public void configureValueConverter(final Map<String, String> config, final AbstractConfig sourceConfig) {
-        config.put(SCHEMA_REGISTRY_URL, sourceConfig.getString(SCHEMA_REGISTRY_URL));
+    public Schema getKeySchema() {
+        return null;
```

Comment:
Do we not need the schema reg here, I thought Parquet did require the schema reg url?